File: PY/president.py
#president

class President:
    
    def __init__(self, name):
        self.name = name
        
    def make_law(self, bill):
        
        while True:
            president_make_law = input("Does President want to sign the bill?(yes/no)").lower()
        
            if president_make_law == "yes":
                print(f"{self.name} signed the bill.")
                return True
            elif president_make_law == "no":
                print(f"{self.name} did not sign the bill.")
                return False
            else:
                print("Invalid input. Please enter 'yes' or 'no'.")
            
    def get_name(self):
        return self.name

File: C#/President.cs

public class President {
    private string Name;


    public President(string Name){
        this.Name = Name;
    }

    public bool makeLaw(Bill bill){
        while(true){
            Console.WriteLine("\n\nDoes president want to sign the bill?");
            string President_MakeLaw;
            President_MakeLaw = Console.ReadLine();
            if(President_MakeLaw.Equals("yes")){
                Console.WriteLine("Bill: \n"+bill.getBill()+"\nMade to Law...");
                return true;
            }
            else if(President_MakeLaw.Equals("no")){
                Console.WriteLine("\nPresident doesnt want to sign so bill cannot be passed....\n");
                return false;
            }
            else{
                Console.WriteLine("Invalid Input");
            }
        }
    }

    public string getName() {
        return Name;
    }
    
}
